feat: add `govc vm.customize` command to support password and custom script (#3867)
vmware/govmomi: govc/USAGE.md

  govc vm.customize -vm VM -auto-login 3 NAME
  govc vm.customize -vm VM -prefix demo NAME
  govc vm.customize -vm VM -tz America/New_York NAME

>   # Change password, expire password, run customization script, specify the vcfa signature

Reviewer: ```suggestion
  # Change password, expire password, run customization script, specify the VCF Automation signature
```

>   govc vm.customize -vm VM -name my-hostname -ip dhcp -pwd 'NewPwd123@' -expire-pwd -script-file './test.bat' -vcfa-sig '<your hash>'

Reviewer: ```suggestion
  govc vm.customize -vm VM -name my-hostname -ip dhcp -pwd '<your password>' -expire-pwd -script-file './test.bat' -vcfa-sig '<your hash>'
```

  -netmask=[]            Netmask
  -org=                  Windows only : name of the org that owns the VM
  -prefix=               Host name generator prefix
  -pwd=                  The new administator (for Windows) or root (for Linux) password
  -script-file=          Path to a script to run before and after customization
  -type=Linux            Customization type if spec NAME is not specified (Linux|Windows)
  -tz=                   Time zone
  -username=             Windows only : full name of the end user in firstname lastname format

>   -vcfa-sig=             The VCFA signanture to identify customization request source

Reviewer: ```suggestion
  -vcfa-sig=             The VCF Automation signature to identify customization request source
```
